lib/internal/fs/promises.js

```diff
@@ -276,12 +275,9 @@ class FileHandle extends EventEmitter {
   /**
    * @typedef {import('../webstreams/readablestream').ReadableStream
    * } ReadableStream
-   * @param {{
-   *   type?: string;
-   *   }} [options]
    * @returns {ReadableStream}
    */
-  readableWebStream(options = kEmptyObject) {
+  readableWebStream() {
```

Comment:
```suggestion
  readableWebStream(options = { type: 'bytes' }) {
```

If we default the `type` to `bytes`, we can emit a warning when the `type` is not `bytes`